txn_writer.go

```diff
@@ -307,22 +312,25 @@ func (sw *SafeWriter) Write(root string, sm gps.SourceManager, noExamples bool)
 			return errors.Wrap(err, "failed to marshal manifest to TOML")
 		}
 
-		var initOutput string
+		var initOutput []byte
 
 		// If examples are NOT disabled, use the example text
 		if !noExamples {
 			initOutput = exampleTOML
 		}
 
-		// 0666 is before umask; mirrors behavior of os.Create (used by
```

Comment:
IMO it's better to leave it as-is. By using `0666`, we're providing the (unix) user full control over the perms with  the files should be written via the dep process' umask. Admittedly, this is coarse-grained, as the umask will affect _all_ writes, including everything involved in populating `vendor`.

If we want to change our approach here to introduce variability, that seems like a larger discussion that probably merits its own issue. How does that sound?